src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,97 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { addCommentPost } from '../api/comments';
+import { Comment } from '../types/Comment';
+
+type Props = {
+  postId: number,
+  onCommentAdd: (comment: Comment) => void,
+  setHasError: (bool: boolean) => void,
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  postId, onCommentAdd, setHasError,
+}) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [body, setBody] = useState('');
+
+  const [nameError, setNameError] = useState(false);
+  const [emailError, setEmailError] = useState(false);
+  const [bodyError, setBodyError] = useState(false);
```

Comment:
Rename boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig). Fix all cases
```suggestion
  const [isNameError, setIsNameError] = useState(false);
  const [isEmailError, setIsEmailError] = useState(false);
  const [isBodyError, setIsBodyError] = useState(false);
```